desktop/faqs.md

```diff
@@ -0,0 +1,213 @@
+
+### How is personal data handled in Docker Desktop?
+
+When uploading diagnostics to help Docker with investigating issues, the uploaded diagnostics bundle may contain personal data such as usernames and IP addresses. The diagnostics bundles are only accessible to Docker, Inc.
+employees who are directly involved in diagnosing Docker Desktop issues.
+
+By default Docker, Inc. will delete uploaded diagnostics bundles after 30 days. You may also request the removal of a diagnostics bundle by either specifying the diagnostics ID or via your GitHub ID (if the diagnostics ID is mentioned in a GitHub issue). Docker, Inc. will only use the data in the diagnostics bundle to investigate specific user issues, but may derive high-level (non personal) metrics such as the rate of issues from it.
+
+## Mac FAQs
+
+### What is Docker.app?
+
+`Docker.app` is Docker Desktop on Mac. It bundles the Docker client and Docker Engine. `Docker.app` uses the macOS Hypervisor.framework to run containers.
+
+### Is Docker Desktop compatible with Apple silicon processors?
+
+At the moment, Docker Desktop is compatible with Intel processors only. For more information, see [Apple M1 Tech preview](../docker-for-mac/apple-m1.md).
+
+You can also follow the status of Apple Silicon support in our [Roadmap](https://github.com/docker/roadmap/issues/142){:target="_blank" rel="noopener" class="_"}.
+
+### What is HyperKit?
+
+HyperKit is a hypervisor built on top of the Hypervisor.framework in macOS. It runs entirely in userspace and has no other dependencies.
+
+We use HyperKit to eliminate the need for other VM products, such as Oracle
+VirtualBox or VMWare Fusion.
+
+### What is the benefit of HyperKit?
+
+HyperKit is thinner than VirtualBox and VMWare fusion, and the version we include is customized for Docker workloads on Mac.
+
+### Why is com.docker.vmnetd still running after I quit the app?
+
+The privileged helper process `com.docker.vmnetd` is started by `launchd` and
+runs in the background. The process does not consume any resources unless
+Docker.app connects to it, so it's safe to ignore.
+
+## Windows FAQs
+
+### Can I use VirtualBox alongside Docker Desktop?
+
+Yes, you can run VirtualBox along with Docker Desktop if you have enabled the [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/){: target="_blank" rel="noopener" class="_"} feature on your machine.
+
+### Why is Windows 10 required?
+
+Docker Desktop uses the Windows Hyper-V features. While older Windows versions have Hyper-V, their Hyper-V implementations lack features critical for Docker Desktop to work.
+
+### Can I install Docker Desktop on Windows 10 Home?
+
+If you are running Windows 10 Home (starting with version 2004), you can install [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows/){: target="_blank" rel="noopener" class="_"} or a later release with the [WSL 2 backend](../docker-for-windows/wsl.md).
```

Comment:
Now 1903.

Reply:
Also delete "or a later release", I think it must be left over from an earlier version of the answer.

Reply:
Fixed